pypesto/ensemble/ensemble.py

```diff
@@ -357,12 +367,15 @@ def _map_parameters_by_objective(self,
 
```

Comment:
If you initialize Ensemble with `x_names` which are not the full parameter vector this leads to `np.nan` entries in the mapping leading to 
```
IndexError: only integers, slices (`:`), ellipsis (`...`), numpy.newaxis (`None`) and integer or boolean arrays are valid indices
``` 
in line 423. I would just drop the `default_value` i.e.
```
 mapping = [
            parameter_ids_ensemble.index(parameter_id_objective)
            for parameter_id_objective in parameter_ids_objective
            if parameter_id_objective in parameter_ids_ensemble
        ]
```
However I might have missed some (potential) use of this `default_value`.

Reply:
@paulstapor Any thoughts on this?

Reply:
yikes... yes, that's a little non-trivial...
Just deleting the non-default value may not work either (have you tested @MerktSimon ?), because the function wants to retrieve the values for all parameter which the objective needs... At least that might/should be a problem...

Reply:
For me it worked with the code I wrote above.